backend/app/routers/render.py

```
@@ -77,9 +67,9 @@ def _render_video_blocking(
         slug_prompt = get_slug_prompt(title=title)
         try:
             emsg = llm.invoke(slug_prompt)
-            file_stem = _slugify_en(emsg.content.strip()) or _slugify_en(title)
+            file_stem = slugify_title(emsg.content.strip()) or slugify_title(title)
         except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)